gcloud-java-pubsub/README.md

```diff
@@ -0,0 +1,25 @@
+Google Cloud Java Client for Pub/Sub
```

Comment:
can you change it based on a recent [README.md](https://github.com/GoogleCloudPlatform/gcloud-java/tree/master/gcloud-java-storage)